server/opts.go

```diff
@@ -255,6 +255,8 @@ type Options struct {
 	// private fields, used for testing
 	gatewaysSolicitDelay time.Duration
 	routeProto           int
+
+	Tags jwt.TagList `json:"-"`
```

Comment:
Let's move this above the private fields. Also, since this is a public option, could you have a little comment? 
```
// Tags <whatever comment>
Tags jwt.TagList `json:"-"`

// private fields, used for testing
...
```

Reply:
fixed